flake-info/default.nix

```diff
@@ -24,7 +24,7 @@ pkgs.rustPlatform.buildRustPackage rec {
 
   ROOTDIR = builtins.placeholder "out";
   NIXPKGS_PANDOC_FILTERS_PATH = pkgs.path + "/doc/build-aux/pandoc-filters";
```

Comment:
This as well.

Reply:
Thanks, done in https://github.com/NixOS/nixos-search/pull/670